@tmeasday CI error

```
lerna ERR! yarn run prepare exited 2 in '@storybook/addon-storyshots'
lerna ERR! yarn run prepare stdout:
$ node ../../../scripts/prepare.js
##[error]src/frameworks/configure.ts(89,41): error TS2554: Expected 2 arguments, but got 3.
```

--- a/addons/storyshots/storyshots-core/src/frameworks/Loader.ts
+++ b/addons/storyshots/storyshots-core/src/frameworks/Loader.ts
@@ -6,7 +6,7 @@ import { SupportedFramework } from './SupportedFramework';
 export type RenderTree = (story: any, context?: any, options?: any) => any;
 
 export interface ClientApi extends ClientStoryApi<unknown> {
-  configure(loader: Loadable, module: NodeModule | false): void;
+  configure(loader: Loadable, module: NodeModule | false, showDeprecationWarning?: boolean): void;
   forceReRender(): void;
   clearDecorators: ClientApiThing['clearDecorators'];
   getStorybook: ClientApiThing['getStorybook'];

